### PR TITLE
bpo-23831: Add moveto method to the tkinter.Canvas widget

### PR DESCRIPTION
**Pull request by j4321**
Add the ``moveto`` method (tk >= 8.6.0) to the ``tkinter.Canvas`` widget.

https://bugs.python.org/issue23831

### CODE REVIEW

File: Lib/tkinter/test/test_tkinter/test_widgets.py [hunk 1]
```diff
@@ -745,6 +745,28 @@ def test_yscrollincrement(self):
         self.checkPixelsParam(widget, 'yscrollincrement',
                               10, 0, 11.2, 13.6, -10, '0.1i')
 
+    def test_moveto(self):
```
Comment (serhiy-storchaka) [inline Lib/tkinter/test/test_tkinter/test_widgets.py:749]: Decorate this test as requiring Tk 8.6.

File: Lib/tkinter/__init__.py [hunk 2]
```diff
@@ -2589,6 +2589,15 @@ def tag_lower(self, *args):
     def move(self, *args):
         """Move an item TAGORID given in ARGS."""
         self.tk.call((self._w, 'move') + args)
+    def moveto(self, tagOrId, xPos, yPos):
```
Comment (serhiy-storchaka) [inline Lib/tkinter/__init__.py]: Please add empty lines around a new method to conform PEP 8. This file is far from conforming PEP 8, but let use PEP 8 for new code.

Comment (serhiy-storchaka) [inline Lib/tkinter/__init__.py]: I wondering whether it is worth to make coordinates optional: `def moveto(self, tagOrId, x='', y='')`.

Then you could specify just *x* or *y* argument: `canvas.moveto('group', y=100)`.

File: Misc/NEWS.d/next/Library/2018-10-09-15-44-04.bpo-23831.2CL7lL.rst [hunk 3]
```diff
@@ -0,0 +1,2 @@
+Add ``moveto`` method to the ``tkinter.Canvas`` widget. Patch by Juliette
```
Comment (serhiy-storchaka) [inline Misc/NEWS.d/next/Library/2018-10-09-15-44-04.bpo-23831.2CL7lL.rst]: It is common to add `()` after the method name: ``` ``moveto()`` ```. The `:meth:` and `:func:` role adds it automatically.

File: Lib/tkinter/__init__.py (hunk 2 above)
Comment (j4321) [reply Lib/tkinter/__init__.py]: Yes, I think it is a good idea, ``canvas.moveto('group', y=100)`` is more natural than passing an empty string to the x argument.

Comment (j4321) [reply Lib/tkinter/__init__.py]: Ok, good to know, I usually conform with PEP 8 but I followed the file style here.

Comment (serhiy-storchaka) [reply Lib/tkinter/__init__.py]: Oh, I wanted to add empty lines just around the new method.

Comment (j4321) [reply Lib/tkinter/__init__.py]: It looked strange to me to have empty lines around one method only so I did the whole class. I can remove the extra ones I added if you want.

Comment (serhiy-storchaka) [reply Lib/tkinter/__init__.py]: Agree, this looks weird. But this is a large change not related to this issue. I have opened a new issue just for adding empty lines: bpo-34964. After merging that change you will need to merge your branch with master.

**Review by serhiy-storchaka — APPROVED** [review]

**Pull request merged**